lib/Signaling/BackendNotifier.php

```diff
@@ -82,6 +84,13 @@ protected function doRequest(string $url, array $params): void {
 			if (!$this->signalingManager->isCompatibleSignalingServer($response)) {
 				throw new \RuntimeException('Signaling server needs to be updated to be compatible with this version of Talk');
 			}
+		} catch (ServerException $e) {
+			if ($retries > 1) {
+				$this->logger->error('Failed to send message to signaling server, ' . $retries . ' retries left!', ['exception' => $e]);
```

Comment:
I don't think all requests should be retried in all cases. E.g. if the shared secret is wrong and an authentication error is returned, it doesn't make much sense to retry the request as the retries will fail, too.

Reply:
ServerException is 5xx status code. 4xx are in ClientException:
https://docs.guzzlephp.org/en/stable/quickstart.html#exceptions

That being said, I guess we should also retry on ConnectException?